packages/lisk-p2p/src/p2p.ts

```diff
@@ -166,50 +166,51 @@ export class P2P extends EventEmitter {
 		});
 	}
 
-	private async _loadListOfPeerListsFromSeeds(
-		seedList: ReadonlyArray<PeerInfo>,
-	): Promise<ReadonlyArray<ReadonlyArray<Peer>>> {
-		return Promise.all(
-			seedList.map(async (seedPeer: PeerInfo) => {
-				const peer = new Peer({
-					ipAddress: seedPeer.ipAddress,
-					wsPort: seedPeer.wsPort,
-					nodeStatus: this._nodeStatus,
-					height: seedPeer.height,
-					os: seedPeer.os,
-					version: seedPeer.version,
-				});
-				this._newPeers.add(peer.peerInfo);
-				/**
-				 * TODO LATER: For the LIP phase, we shouldn't add the seed peers to our
-				 * _peerPool and we should disconnect from them as soon as we've loaded their peer lists.
-				 */
-				this._peerPool.addPeer(peer);
-				// TODO LATER: For the LIP phase, the 'list' request will need to be renamed.
-				const seedNodePeerListResponse = await peer.request<void>({
-					procedure: 'list',
-				});
-				const peerListResponse = seedNodePeerListResponse.data as ProtocolPeerList;
-				// TODO ASAP: Validate the response before returning. Check that seedNodePeerListResponse.data.peers exists.
-
-				return peerListResponse.peers.map(
-					(peerObject: ProtocolPeerInfo) =>
-						new Peer({
-							ipAddress: peerObject.ip,
-							wsPort: peerObject.wsPort, // TODO ASAP: Add more properties
-							nodeStatus: this._nodeStatus,
-							height: seedPeer.height,
-							os: seedPeer.os,
-							version: seedPeer.version,
-						}),
-				);
-			}),
+	private _getPeerToConnect(): ReadonlyArray<Peer> {
```

Comment:
can it be pure function?

Reply:
yeah it can be or maybe we can also merge it inside `_connectToPeers()` ?

Reply:
I merged this function inside `_connectToPeers()` because it was a bit unnecessary to make a separate function to call selection algo